pages/index.tsx

```diff
@@ -30,7 +30,7 @@ for await (const req of s) {
       </Head>
       <CookieBanner />
       <div className="bg-white">
-        <div className="bg-gray-50 border-b border-gray-200">
+        <div className="bg-gray-50 border border-b border-gray-200">
```

Comment:
`border` is `border-width: 1px;`, `border-b` is `border-bottom-width: 1px;`. Specifying both doesn't make sense. (`border-b` is included in `border`)

Reply:
Sometimes tailwind doesn't render borders so its best to specify `border` seperately along with the classes